### PR TITLE
jormungandr:correction in links of journeys api

### PR DESCRIPTION
**Pull request by azime**
in new_default, prev link for clockwise request is buggy. As next link in anticlockwise request.

This fix this issue.

fix http://jira.canaltp.fr/browse/ITI-23

### CODE REVIEW

File: source/jormungandr/jormungandr/scenarios/new_default.py [hunk 1]
```diff
@@ -598,6 +598,7 @@ class Scenario(simple.Scenario):
     def __init__(self):
         super(Scenario, self).__init__()
         self.nb_kraken_calls = 0
+        self.clockwise = True
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/scenarios/new_default.py]: I'm not sure that the instances are by request.

File: source/jormungandr/jormungandr/scenarios/new_default.py [hunk 2]
```diff
@@ -694,10 +695,11 @@ def create_next_kraken_request(self, request, responses):
 
         to do that we find ask the next (resp previous) query datetime
         """
+        vjs = [j for r in responses for j in r.journeys if not journey_filter.to_be_deleted(j)]
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/scenarios/new_default.py:696]: :+1:

File: source/jormungandr/tests/check_utils.py [hunk 3]
```diff
@@ -1117,9 +1117,10 @@ def make_crit(func, reverse=False):
         return partial(reverse_compare_field, func=func)
 
     main_criteria = make_crit(lambda j: get_valid_datetime(j['departure_date_time'])) if clockwise \
-        else make_crit(lambda j: get_valid_datetime(j['arrival_date_time']), reverse=True)
+        else make_crit(lambda j: get_valid_datetime(j['arrival_date_time']))
```
Comment (TeXitoi) [inline source/jormungandr/tests/check_utils.py]: I doubt this modification is valid

**Comment by TeXitoi** [comment]
@antoine-de @kinnou02 @pbougue can one of you review this PR? thanks.

File: source/jormungandr/jormungandr/scenarios/new_default.py (hunk 1 above)
Comment (kinnou02) [inline source/jormungandr/jormungandr/scenarios/new_default.py]: Nop, there is only one global scenariouse by all jormungandr threads

**Comment by kinnou02** [comment]
A explanation about what the problem is would be great

**Comment by TeXitoi** [comment]
Description updated.

**Comment by TeXitoi** [comment]
OK for me. @kinnou02 ?

**Comment by kinnou02** [comment]
Ok for me too

**Pull request merged**